cmd/algons/dnsCmd.go

```diff
@@ -268,7 +268,8 @@ func checkSrvRecord(dnsBootstrap string) {
 
 func doDeleteDNS(network string, noPrompt bool, excludePattern string) bool {
 
-	if network == "" || network == "testnet" || network == "devnet" || network == "mainnet" {
+	validNetworks := map[string]bool{"mainnet": true, "testnet": true, "devnet": true, "betanet": true}
```

Comment:
if we're not going to make these common (here and cmd/goal/node.go ) can we at least put a comment to keep the other one up-to-date with this one (in both places) ?